redwood-lazylayout-compose/src/commonTest/kotlin/app/cash/redwood/lazylayout/compose/LazyListTest.kt

```diff
@@ -78,20 +68,53 @@ class LazyListTest {
 
     assertThat(snapshot)
       .containsExactly(
-        LazyListValue(
-          Modifier,
-          isVertical = true,
-          onViewportChanged = { _, _ -> },
-          itemsBefore = 0,
-          itemsAfter = 0,
-          width = Constraint.Wrap,
-          height = Constraint.Wrap,
-          margin = Margin(0.0.dp),
-          crossAxisAlignment = CrossAxisAlignment.Start,
-          scrollItemIndex = ScrollItemIndex(0, 0),
+        DefaultLazyListValue.copy(
           placeholder = List(20) { TextValue(Modifier, "Placeholder") },
           items = List(10) { TextValue(Modifier, it.toString()) },
         ),
       )
   }
 }
+
+private val DefaultLazyListValue = LazyListValue(
+  Modifier,
+  isVertical = true,
+  onViewportChanged = { _, _ -> },
+  itemsBefore = 0,
+  itemsAfter = 0,
+  width = Constraint.Wrap,
+  height = Constraint.Wrap,
+  margin = Margin(0.0.dp),
+  crossAxisAlignment = CrossAxisAlignment.Start,
+  scrollItemIndex = ScrollItemIndex(0, 0),
+  placeholder = emptyList(),
+  items = emptyList(),
+)
+
+private fun LazyListValue.copy(
```

Comment:
Ideally, this'd be generated within `LazyListValue`.

Reply:
I'm guessing we don't want `copy` to be generated, as it'd need to be part of the public API which [we don't want](https://github.com/cashapp/redwood/issues/1043). Though these `WidgetValue` instances are so far exclusively used in tests, so maybe that doesn't matter?

Reply:
Test API definitely has more flexibility, although breaking it could still break people.